updatecli/values.d/update-cli-ref-docs.yaml

```diff
@@ -0,0 +1,26 @@
+---
+# Values.yaml contains settings that be used from Updatecli manifest.
+sourceFiles: 
+  - "cli-docs.md"
+targetFiles: 
+  - "docs/reference/kwctl-cli.md"
```

Comment:
Is the copied file `kwctl-cli.md` consumable by the docs directly? They may need a header, and this header may change in the future if we move away from docusaurus.